functions/notify_slack.py

```diff
@@ -97,8 +97,10 @@ def notify_slack(subject, message, region):
     payload['text'] = "AWS notification"
     payload['attachments'].append(default_notification(subject, message))
 
-  data = urllib.parse.urlencode({"payload": json.dumps(payload)}).encode("utf-8")
+  data = json.dumps(payload).encode('utf-8')
```

Comment:
`payload` key is missing, right?

Reply:
Also, the lack of urlencode suddenly, will this break something?  Was/is this not needed with the new headers set?

Reply:
@antonbabenko In spite of it works with `payload` as root key, this isn't clearly specified in the documentation. 
https://api.slack.com/messaging/webhooks

Reply:
Maybe `Content-Length` might not be really necessary, as urlib documentation states that is set when data is sent. 

On other hand, when `Content-Type` is not informed, it is added as `application/x-www-form-urlencoded` as default, which diverges of `application/json` required by Slack

https://docs.python.org/3/library/urllib.request.html#urllib.request.Request